docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Edit Kubernetes overrides
+headerTitle: Edit Kubernetes overrides
+linkTitle: Edit Kubernetes overrides
+description: Edit Kubernetes overrides for a universe.
+aliases:
+  - /preview/manage/enterprise-edition/edit-helm-overrides/
+menu:
+  preview_yugabyte-platform:
```

Comment:
Need to fix menu this duplicates the old page

Reply:
I don't know what you mean. The menu looks fine in the build preview